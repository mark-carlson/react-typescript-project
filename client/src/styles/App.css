body {
  background-color: #f8f8f8;
  font-family: var(--mainFontFamily), sans-serif;
  margin: 0;
}

.block {
  width: 100%;
}

.overflow-hidden 
.model-hide-overflow {
  overflow: hidden;
}

.hidden {
  display: none;
}

.overflow-hidden {
  overflow: hidden;
}

.visibility-hidden {
  visibility: hidden;
}

.is-clickable {
  cursor: pointer;
}

.centered {
  text-align: center;
  margin-left: auto;
  margin-right: auto;
}