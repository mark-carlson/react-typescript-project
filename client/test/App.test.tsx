import React from 'react';
import renderer from 'react-test-renderer';
import App from 'App';

describe('App', () => {
    it('Shows Hello World initially', (): void => {
        const app = renderer.create(<App />);

        expect(app).toMatchSnapshot();
    });
});
