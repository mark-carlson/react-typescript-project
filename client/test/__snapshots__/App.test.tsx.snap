// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App Shows Hello World initially 1`] = `
<div>
  Hello World!!
</div>
`;
